hollow/src/main/java/com/netflix/hollow/tools/stringifier/HollowRecordJsonStringifier.java

```diff
@@ -78,83 +78,67 @@ public HollowRecordJsonStringifier addExcludeObjectTypes(String... types) {
         return this;
     }
 
-
-    /**
-     * Create a JSON representation of the provided {@link HollowRecord}.
-     */
```

Comment:
Removed duplicate javadocs - the interface method that we override has the javadocs, no sense in duplicating them